pyomo/core/base/constraint.py

```diff
@@ -469,14 +466,14 @@ def set_value(self, expr):
                         "Constraint '%s' does not have a proper "
                         "value. Found '%s'\nExpecting a tuple or "
                         "equation. Examples:"
-                        "\n   summation(model.costs) == model.income"
+                        "\n   sum_product(model.costs) == model.income"
```

Comment:
This is not a good example, if for no other reason than there is no product being formed.  I would rather examples like:
```
sum_product(model.unit_price, model.sales) == model.income
sum(model.costs[i] for i in model.costs) == model.expenses  # or quicksum
```